### ISSUE TITLE
Check for h1 tag

### ISSUE THREAD

**Issue by BarryBotha** [passmarked/seo#7]
Each page should have an h1 tag as the main heading. Subsequent h1 tags must be within a sectioning  root or content sectioning tags.

Read more: [https://www.w3.org/TR/html5/sections.html](https://www.w3.org/TR/html5/sections.html)

**Comment by Johanndutoit** [comment]
Had a read, bit confused on how we would separate sections. Any ideas ? Will do some research and get back to you, @passmarked/open-source-devs the team might have some ideas too 🔥

**Comment by BarryBotha** [comment]
This is a good read about the topic: [https://webdesign.tutsplus.com/articles/the-truth-about-multiple-h1-tags-in-the-html5-era--webdesign-16824](https://webdesign.tutsplus.com/articles/the-truth-about-multiple-h1-tags-in-the-html5-era--webdesign-16824)

**Comment by Johanndutoit** [comment]
Right, that helped quite a bit. Looking at ways to build a tree that we can check in a rule now. Will report back on what I find. 👍

**Comment by Johanndutoit** [comment]
mmm, still looking into this one
